README.md

```diff
@@ -164,11 +164,11 @@ threads are handled by Android runtime.
 
 #### R8 and ProGuard
 
-For R8 no actions required, it will take obfuscation rules from the jar.
-
-For Proguard  you need to add options from [coroutines.pro](kotlinx-coroutines-core/jvm/resources/META-INF/proguard/coroutines.pro) to your rules manually.
- 
-R8 is a replacement for ProGuard in Android ecosystem, it is enabled by default since Android gradle plugin 3.4.0 (3.3.0-beta also had it enabled).
+R8 and ProGuard rules are bundled into the [`kotlinx-coroutines-android`](ui/kotlinx-coroutines-android) module.
+For best results it is recommended to use a recent version of R8.
+R8 is a replacement for ProGuard in Android ecosystem, it is enabled by default since Android gradle plugin 3.4.0
+(3.3.0-beta also had it enabled). The upcoming AGP 4.0.0 has never R8 and additional rules enable
```

Comment:
```suggestion
(3.3.0-beta also had it enabled). The upcoming AGP 4.0.0 has newer R8 and additional rules enable
```
?

Reply:
BTW @wojtek-kalicinski have fixed the version checks in AGP 3.6 too, so this works starting AGP 3.6.0

Reply:
Also: enable -> enabled (end of the sentence)

Reply:
```suggestion
(3.3.0-beta also had it enabled). The upcoming AGP 4.0.0 has newer R8 and additional rules enabled
```

Reply:
👍